package.json

```diff
@@ -33,10 +34,14 @@
     ]
   },
   "devDependencies": {
+    "@swc/core": "^1.2.135",
+    "@swc/helpers": "^0.3.2",
```

Comment:
Enables fast transpilation in `ts-node`
https://github.com/TypeStrong/ts-node#bundled-swc-integration

Reply:
SWC :rocket: